code/models/rapids/train_rapids.py

```diff
@@ -0,0 +1,130 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import os
+import time
+
+import numpy as np
+import pandas as pd
+import cudf
+import cuml
+
+from cuml import RandomForestClassifier as cuRF
+from cuml.preprocessing.model_selection import train_test_split
+from cuml.metrics.accuracy import accuracy_score
+
+from rapids_csp_azure import RapidsCloudML, PerfTimer
+from azureml.core.run import Run
+
+run = Run.get_context()
```

Comment:
no azureml-* code allowed in training scripts - see contributing guidelines - log via mlflow